### PR TITLE
Wire Behaviour UI fixes

### PR DESCRIPTION
**Pull request by M-JULIANI**
### Description 
UI fixes addressing some of the recent deviations in expected behaviour:
- Button rectangular border removed from connector hover icon buttons (watch/ pin/ anchor).
- Connector tooltip behaviour to match that of temporary hover icons.
- Hover state for polyline connectors disabled.
- Instantiation location of pins fixed.

*Node layout regression test fixes will be addressed in a different PR.

WireBehaviour-UI Fixes PR

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers
@QilongTang 

### FYIs
@SHKnudsen 
@Amoursol

### CODE REVIEW

File: src/DynamoCore/Graph/ConnectorPinModel.cs [hunk 1]
```diff
@@ -7,12 +7,17 @@ namespace Dynamo.Graph
...
         public Guid ConnectorId { get; set; }
+        /// <summary>
+        /// Required when needing to pull a reliable value for objects of this
+        /// type when there is no instances of this object type.
+        /// </summary>
+        public const double StaticWidth = width;
 
         public override double Height { get => height; }
-        public override double Width { get => width; }
+        public override double Width { get => StaticWidth; }
```
Comment (QilongTang) [inline src/DynamoCore/Graph/ConnectorPinModel.cs:20]: What's the purpose of this change? To me seems no need to change?

**Review by QilongTang — APPROVED** [review]
One comment then LGTM

File: src/DynamoCore/Graph/ConnectorPinModel.cs (hunk 1 above)
Comment (M-JULIANI) [reply src/DynamoCore/Graph/ConnectorPinModel.cs:20]: Honestly just setting a `const` there so that I can access a value corresponding the the class without there needing to be an instance of the class. Perhaps it is best if I make it a `static` property instead? On line 20 just making use of that variable as opposed to pointing to yet another thing, but agreed I can reveft line 15 to what it was previously.

**Comment by QilongTang** [comment]
@M-JULIANI Can you take a look at the current state? I think the PR does not build for some reason

**Comment by M-JULIANI** [comment]
@QilongTang  having a hard time replicating that behaviour. It is building fine for me?

**Comment by M-JULIANI** [comment]
@QilongTang  ah you're right, one sec.

**Comment by QilongTang** [comment]
hmm still failed from the first Github action, are you using a newer C# version or?

**Comment by M-JULIANI** [comment]
image
No should be using correct one.

**Comment by QilongTang** [comment]
Well, that's strange, I will merge and give it a try locally

**Comment by M-JULIANI** [comment]
@QilongTang  lmk if you find something unusual!

**Comment by QilongTang** [comment]
@M-JULIANI After merging I found your changes to connectorPinModel are all gone.. That was the reason why，but I dont understand the reason yet :)

**Comment by M-JULIANI** [comment]
Hmm nor do I

**Comment by M-JULIANI** [comment]
How is that _gone_ if it appears in the commit history & files changed?

**Comment by M-JULIANI** [comment]
Yeah but I am seeing the same thing when I checkout the newly merged branch... @QilongTang

**Comment by QilongTang** [comment]
@M-JULIANI Should be fixed as part of https://github.com/DynamoDS/Dynamo/pull/11952

**Comment by M-JULIANI** [comment]
Great!

**Comment by M-JULIANI** [comment]
@QilongTang  not sure if there is a lag or something, but when I checkout newly merged `DynamoNodeRedesign` I'm not seeing the changes reflected. Are you?

**Comment by QilongTang** [comment]
Seeing same @M-JULIANI

**Comment by M-JULIANI** [comment]
Can you confirm that your are seeing `        public static double StaticWidth { get; } = 30;` on line 24 when you check out the newly merged branch? @QilongTang

**Comment by QilongTang** [comment]
@M-JULIANI Yes, I even build and tested
image

**Comment by M-JULIANI** [comment]
Ok it's my issue then ¯\_(ツ)_/¯

**Comment by QilongTang** [comment]
I found out why, in this PR you were modifying a file which should not exist, ConnectorPinModel should be under `Graph/Connectors` but not under `Graph` folder
